modules/Bio/EnsEMBL/Compara/RunnableDB/DumpMemberSequencesIntoFasta.pm

```diff
@@ -85,7 +85,7 @@ sub run {
 
     close FASTAFILE;
 
-    my $n_seq_in_file = `grep -c "^>" "$fasta_name"`;
+    my $n_seq_in_file = $self->run_command(['grep', '-c', '^>', $fasta_name])->out;
```

Comment:
I'm a bit unsure about how `grep` will handle the `'^>'` here - I think we should add double quotes around it to be safe, as it was in the original command

Reply:
I will double-check, but with the array form, each element becomes a separate argument as is. I think if we use `'">"'`, `grep` would get `"^>"` as pattern and would be looking for double-quotes in the fasta file

Reply:
Confirmed: it doesn't work with the double quotes